web/custom-plugins/forecast-of-response/src/Types.ts

```diff
@@ -0,0 +1,41 @@
+export type TRoute = {
+  path: string
+  content: string
+}
+
+export type TSettings = {
+  name: string
+  label: string
+  tabIndex: number
+  hidden: boolean
+  visibleDataSources: any
+  type: string
+  description: string
+  packages: any
+  models: any
+  actions: any
+  file_loc: string
+  data_source_aliases: any
+}
```

Comment:
Good stuff! But this is DMT specific. And should be moved to the DMT package and imported from there.... in some way.

Reply:
@soofstad How about adding it to the "types.tsx" in dmt @ web/plugins/core-plugins/src/types.tsx?

Reply:
Added a TODO for now.

Reply:
Added a PR in equinor/data-modelling-tool/pull/907